common/Strings/en-us/Resources.resw

```diff
@@ -323,4 +323,40 @@
   <data name="ChooseFileActionToolTip" xml:space="preserve">
     <value>Choose file</value>
   </data>
+  <data name="ContainersDescription" xml:space="preserve">
+    <value>Provides services and tools to create and manage Windows Server Containers and their resources.</value>
+    <comment>Description for the Containers optional feature.</comment>
+  </data>
+  <data name="GuardedHostDescription" xml:space="preserve">
+    <value>Enables the device to create and run Shielded Virtual Machine using remote attestation.</value>

```

Comment:
Going to just lock Shielded Virtual Machine since that's the product name associated with the feature. These descriptions are taken from DISM so waiting on the content review before spending too much time on these. Same generally goes for the other strings changes here.